### ISSUE TITLE
Add support for VS 2019

### ISSUE THREAD

**Issue by madskristensen** [Microsoft/slow-cheetah#160]
It's super easy and quick to do. See how here https://madskristensen.net/blog/how-to-upgrade-extensions-to-support-visual-studio-2019/

**Comment by acehenry** [comment]
Yes please!  Love this tool and would love to continue using it in 2019!

**Comment by Rzpeg** [comment]
Please add VS2019 support!
